controller_report_xls/i18n/es_ES.po

```diff
@@ -0,0 +1,15 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
```

Comment:
The file `es.po` should be regenerated as part of the migration. Besides, remove file `es_ES.po` and similar ones in the previous version.

Reply:
File was not regenerated in this version, still pointing to 15.0.